nipype/interfaces/freesurfer/petsurfer.py

```diff
@@ -0,0 +1,651 @@
+class MRTM(GLMFit):
+    """Perform MRTM1 kinetic modeling.
+    Examples
+    --------
+    >>> mrtm = MRTM()
+    >>> mrtm.inputs.in_file = 'tac.nii'
+    >>> mrtm.inputs.mrtm1 = [('ref_tac.dat', 'timing.dat')]
+    >>> mrtm.inputs.glmdir = 'mrtm'
+    >>> mrtm.cmdline == 'mri_glmfit --glmdir mrtm --y tac.nii --mrtm1 ref_tac.dat timing.dat'
+    """
+
+    _cmd = "mri_glmfit"
+    input_spec = MRTMInputSpec
+    output_spec = MRTMOutputSpec
+
+    def _list_outputs(self):
+        outputs = self.output_spec().get()
+        outputs['k2p'] = os.path.join(self.inputs.glm_dir, 'k2prime.dat')
+        return outputs
+
+class MRTM2InputSpec(GLMFitInputSpec):
+    
+    mrtm2 = InputMultiPath(
+        traits.Tuple(File(exists=True), File(exists=True), traits.Float),
+        mandatory=True,
+        argstr="--mrtm2 %s %s %f...",
+        desc="RefTac TimeSec k2prime : perform MRTM2 kinetic modeling", 
+    )
+
+    _ext_xor = ['nii', 'nii_gz']
+    nii = traits.Bool(
+        argstr='--nii',
+        desc='save outputs as nii',
+        xor=_ext_xor
+    )
+    nii_gz = traits.Bool(
+        argstr='--nii.gz',
+        desc='save outputs as nii.gz',
+        xor=_ext_xor
+    )
+    yhat_save = traits.Bool(
+        argstr="--save-yhat", desc="save signal estimate (yhat)"
+    )
+
+class MRTM2OutputSpec(GLMFitInputSpec):
+    bp = File(desc="BP estimates")
+    yhat = File(desc="Fitted values (yhat) to the TACs")
+
+class MRTM2(GLMFit):
+    """Perform MRTM2 kinetic modeling.
+    Examples
+    --------
+    >>> mrtm2 = MRTM2()
+    >>> mrtm2.inputs.in_file = 'tac.nii'
+    >>> mrtm2.inputs.mrtm2 = [('ref_tac.dat', 'timing.dat', 0.07872)]
+    >>> mrtm2.inputs.glmdir = 'mrtm2'
+    >>> mrtm2.cmdline == 'mri_glmfit --glmdir mrtm2 --y tac.nii --mrtm2 ref_tac.dat timing.dat 0.07872'
+    """
+
+    _cmd = "mri_glmfit"
+    input_spec = MRTM2InputSpec
+    output_spec = MRTM2OutputSpec
+
+    def _list_outputs(self):
+        outputs = self.output_spec().get()
+        if isdefined(self.inputs.nii_gz):
+            ext = '.nii.gz'
+        if isdefined(self.inputs.nii):
+            ext = '.nii'
+        else:
+            ext = '.mgh'            
+        outputs['bp'] = os.path.join(self.inputs.glm_dir, 'bp',  ext)
+        outputs['yhat'] = os.path.join(self.inputs.glm_dir, 'yhat',  ext)
+        return outputs
+
+class LoganRefInputSpec(GLMFitInputSpec):
+    logan = InputMultiPath(
+        traits.Tuple(File(exists=True), File(exists=True), traits.Float),
+        mandatory=True,
+        argstr="--logan %s %s %f...",
+        desc="RefTac TimeSec tstar   : perform Logan kinetic modeling", 
+    )
+
+    _ext_xor = ['nii', 'nii_gz']
+    nii = traits.Bool(
+        argstr='--nii',
+        desc='save outputs as nii',
+        xor=_ext_xor
+    )
+    nii_gz = traits.Bool(
+        argstr='--nii.gz',
+        desc='save outputs as nii.gz',
+        xor=_ext_xor
+    )
+
+class LoganRefOutputSpec(GLMFitInputSpec):
+    bp = File(desc="BP estimates")
```

Comment:
If it's a file we generally use `_file` to make things clearer.

```suggestion
    bp_file = File(desc="BP estimates")
```